src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportControlLoader.java

```diff
@@ -227,6 +251,26 @@ private ImportControl getRoot() {
     }
 
     /**
+     * Utility to get a strategyOnMismatch property.
+     * @param attributes collect to get attribute from.
+     * @return the value of the attribute.
+     */
+    private static MismatchStrategy getStrategyOnMismatch(final Attributes attributes) {
+        final String returnValue = attributes.getValue(STRATEGY_ON_MISMATCH_ATTRIBUTE_NAME);
+        MismatchStrategy strategyOnMismatch = null;
+        if ("delegateToParent".equals(returnValue)) {
+            strategyOnMismatch = MismatchStrategy.DELEGATE_TO_PARENT;
+        }
+        else if ("allowed".equals(returnValue)) {
+            strategyOnMismatch = MismatchStrategy.ALLOWED;
+        }
+        else if ("disallowed".equals(returnValue)) {
+            strategyOnMismatch = MismatchStrategy.DISALLOWED;
+        }
+        return strategyOnMismatch;
```

Comment:
I made this way because this method can be applied for both "import-control" and "subpackage" property. So if NULL is not allowed, I will have to make two separate methods for "import-control" and "subpackage", because they have different default strategies.

Also in the beginning of this class I have if condition

```
if (strategyOnMismatch == null) {
                 stack.push(new ImportControl(pkg, regex));
 }
 else {
                 stack.push(new ImportControl(pkg, regex, strategyOnMismatch));
}
```
and inside ImportControl.java I have constructor

```
ImportControl(final String pkgName, final boolean regex) {
        this(pkgName, regex, MismatchStrategy.DISALLOWED);
}
```
As you can see NULL object will not pass to the ImportControl object and default strategy will be initialized with proper value.
I can rewrite code, but I don't understand why my solution is not correct